source/mainnet/net/references/grpc.rst

```diff
@@ -0,0 +1,886 @@
+
+      ``Parameter``: Parameter for the receive function.
+
+      .. code-block::
+
+         ContractAddress ::= (index: UInt64) (subindex: UInt64)
+
+         ReceiveName ::= (n: UInt16) (utf8: Byteⁿ)
+
+         Parameter ::= (n: UInt16) (data: Byteⁿ)
+
+         Update ::= (amount: UInt64) (contractAddress: ContractAddress) (receiveName: ReceiveName) (parameter: Parameter)
+
+   .. _grpc-transfer:
+
+   ``Transfer``
+      Transfer CCD from the sender account to the specified account address.
+      The ``amount`` is microCCD (``10^-6 CCD``).
+
+      .. code-block::
+
+         Transfer ::= (accountAddress: Byte³²) (amount: UInt64)
+
+
+   .. _grpc-transfer-with-memo:
+
+   ``TransferWithMemo``
+      Transfer CCD from the sender account the specified account address and
+      include a memo. The memo can be up to 256 bytes long, excluding its
+      length (``n``).
+
+      .. code-block::
+
+         Memo ::= (n: UInt16) (data: Byteⁿ)
+
+         TransferWithMemo ::= (accountAddress: Byte³²) (memo: Memo) (amount: UInt64)
+
+.. |grpc-block-hash| replace:: ``BlockHash``
+.. |grpc-block-height| replace:: ``BlockHeight``
+.. |grpc-transaction-hash| replace:: ``TransactionHash``
+.. |grpc-account-address| replace:: ``AccountAddress``
+.. |grpc-contract-address| replace:: ``ContractAddress``
+.. |grpc-contract-context| replace:: ``ContractContext``
+.. |grpc-block-item| replace:: ``BlockItem``
+.. |grpc-account-transaction| replace:: ``AccountTransaction``
+.. |grpc-transaction-signature| replace:: ``TransactionSignature``
+.. |grpc-transaction-header| replace:: ``TransactionHeader``
+.. |grpc-transaction-payload| replace:: ``TransactionPayload``
+.. |grpc-module-deploy| replace:: ``ModuleDeploy``
+.. |grpc-init-contract| replace:: ``InitContract``
+.. |grpc-update| replace:: ``Update``
+.. |grpc-get-instances| replace:: ``GetInstances``
+.. |grpc-get-instance-info| replace:: ``GetInstanceInfo``
+.. |grpc-module-reference| replace:: ``ModuleReference``
+.. _NodeInfoResponse: https://github.com/Concordium/concordium-grpc-api/blob/44e9c5825b1b18d9e81d15db30546316aa5906ec/concordium_p2p_rpc.proto#L67
+.. |NodeInfoResponse| replace:: ``NodeInfoResponse``
+.. _BlockHeight: _https://github.com/Concordium/concordium-grpc-api/blob/44e9c5825b1b18d9e81d15db30546316aa5906ec/concordium_p2p_rpc.proto#L146
+.. |BlockHeight| replace:: ``BlockHeight``
+.. _PeerElement: https://github.com/Concordium/concordium-grpc-api/blob/44e9c5825b1b18d9e81d15db30546316aa5906ec/concordium_p2p_rpc.proto#L34
+.. |PeerElement| replace:: ``PeerElement``
+.. _PeerStatsResponse: https://github.com/Concordium/concordium-grpc-api/blob/44e9c5825b1b18d9e81d15db30546316aa5906ec/concordium_p2p_rpc.proto#L51
+.. |PeerStatsResponse| replace:: ``PeerStatsResponse``
+.. _PeerListResponse: https://github.com/Concordium/concordium-grpc-api/blob/44e9c5825b1b18d9e81d15db30546316aa5906ec/concordium_p2p_rpc.proto#L46
+.. |PeerListResponse| replace:: ``PeerListResponse``
```

Comment:
I think these links need to be updated to point to the more recent commit.

Reply:
Yep, I'll do that once the other PR is merged.

Reply:
Done